src/stactools/hotosm/maxar/stac.py

```diff
@@ -0,0 +1,141 @@
+"""Create STAC records for HOT OSM from Maxar's public catalog."""
+
+import datetime as dt
+
+from pystac import (
+    Catalog,
+    Collection,
+    Extent,
+    Item,
+    Link,
+    MediaType,
+    RelType,
+    SpatialExtent,
+    TemporalExtent,
+)
+from pystac.extensions.item_assets import ItemAssetDefinition
+from pystac.extensions.render import Render, RenderExtension
+
+from stactools.hotosm.stac_common import add_alternate_assets
+
+COLLECTION_ID = "maxar-opendata"
+COLLECTION_DESCRIPTION = (
+    "Maxar Open Data Catalog, formatted for Humanitarian OpenStreetMap "
+    "Team's OpenAerialMap project"
+)
+
+
+def create_collection(catalog: Catalog, event_dates: list[dt.datetime]) -> Collection:
```

Comment:
I get the value in setting the initial temporal extent, but a user might want to create the collection without having pre-fetched the dates?

```suggestion
DEFAULT_INTERVAL = [None, None]

def create_collection(catalog: Catalog, event_dates: list[dt.datetime] = DEFAULT_ITERVAL) -> Collection:
```

Reply:
+1